mqemitter-redis.js

```diff
@@ -139,10 +140,11 @@ MQEmitterRedis.prototype.emit = function (msg, done) {
   }
 
   const p = this._pipeline.publish(msg.topic, msgpack.encode(packet))
+
   if (done) {
-    p.then(done, done)
+    p.then(() => done()).catch(done)
   } else {
-    p.catch(noop)
+    p.catch(this._onError.bind(this))
```

Comment:
we should probably maintain a bound `_onError`, as this would be called a lot.

Reply:
So just `this._onError` here?

Reply:
yes, but you'd need to to `this._onError = this._onError.bind(this)`